code/__DEFINES/food.dm

```diff
@@ -14,6 +14,7 @@
 #define BREAKFAST (1<<13)
 #define CLOTH (1<<14)
 #define NUTS (1<<15)
+#define SEAFOOD (1<<16)
 
 /// A list of food type names, in order of their flags
 #define FOOD_FLAGS list( \
```

Comment:
Add SEAFOOD (and NUTS, so it's in order) here

Reply:
Implemented this. I have no idea how to re-request a review for this so I'm hoping this comment gives a notification of some sort.